### DISCUSSION TITLE
battery status

### DISCUSSION THREAD

**Discussion by guestguestguestguest** [mrlt8/docker-wyze-bridge#1156] [Ideas]
New feature request: display battery status for battery-powered cams such as the outdoor model on the Info screen.  There's a class K10448GetBatteryUsage in the tutk_protocol which maybe could be used?  Although it doesn't appear to include the charge percentage, alas.

**Comment by mrlt8** [comment]
I believe K10448 is already accessible via the `battery_usage` api command/mqtt topic. You should also be able to get the battery value from the wyze web-api by using the `battery` api command/mqtt topic.

```
http://10.0.0.2:5000/api/cam-name/battery_usage
http://10.0.0.2:5000/api/cam-name/battery
```

you can check the wiki for additional info: https://github.com/mrlt8/docker-wyze-bridge/wiki/Camera-Commands

**Reply by guestguestguestguest** [reply]
API command battery works, thanks!  Command battery_usage throws error "the JSON object must be str, bytes or bytearray, not NoneType".

**Reply by mrlt8** [reply]
Which battery cam is this? `K10448GetBatteryUsage` seems to work for the outdoor cam.

**Reply by guestguestguestguest** [reply]
firmware_ver	4.17.4.418
model_name	Outdoor
product_model	WVOD1
